config/samples/_v1alpha1_orchestrator_rc.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: rhdh.redhat.com/v1alpha1
+kind: Orchestrator
+metadata:
+  name: orchestrator-sample
+spec:
+  sonataflow:
+    images:
+      4.13: brew.registry.redhat.io/rh-osbs/iib:826480
+      4.14: brew.registry.redhat.io/rh-osbs/iib:826493
+      4.15: brew.registry.redhat.io/rh-osbs/iib:826482
+      4.16: brew.registry.redhat.io/rh-osbs/iib:826483
+      4.17: brew.registry.redhat.io/rh-osbs/iib:826485
```

Comment:
We don't support 4.17

Reply:
I know but we did not support 4.16 but the value was there already so I decided to put the 4.17 as it is now released
As RC files are aimed for ppl who know what they are doing or want to test something, I thought it was not a bad idea to have the latest release here as well
I can remove if you think it's better without 4.17